stdlib/2/_threading_local.pyi

```diff
@@ -0,0 +1,16 @@
+# Source: https://hg.python.org/cpython/file/2.7/Lib/_threading_local.py
+from typing import Any, List, Type, TypeVar
+
+__all__: List[str]
+_cls = TypeVar('_cls')
+
+class _localbase:
+    def __new__(cls: Type[_cls], *args, **kw) -> _cls: ...
```

Comment:
I'd rather omit this method, since it's basically the same as `object.__new__`, and having this override might confuse some type checkers.